h2o-algos/src/main/java/hex/pca/PCA.java

```diff
@@ -457,8 +451,11 @@ public void computeImpl() {
           model._output._validation_metrics = ModelMetrics.getFromDKV(model,_parms.valid());
         }
         model.update(_job);
-
-
+  
+  
+      } catch (Exception e) {
+        e.printStackTrace();
```

Comment:
Pls remove.

Reply:
@mmalohlava Do you mean the print or the throw here?

Reply:
The print should be removed, since the print is going directly into stderr (it is mostly some debugging relic). 

If we need to log the exception, we should use `water.Log.{warn,debug,info}`.